docs/en/wallet-solution.rst

```diff
@@ -2,56 +2,83 @@
 
 .. _wallet-solution.rst:
 
-wallet-solution.rst
+Wallet Solution
 +++++++++++++++++++++++++++
```

Comment:
please remove the exceedings `+`

yes, it's a bane ...

Reply:
@peppelinux are we changing the file paths?

Reply:
done in https://github.com/italia/eidas-it-wallet-docs/pull/15/commits/c790ffdd0eca8eeeec2396e1b0254405b8d4ade5